### PR TITLE
Fix Jumping Buttons

### LINKED ISSUE #12: Buttons jump around when skipping songs
The sound notification indicator should ellipsize song names and always have a fixed witdth to avoid having the buttons jumping around when skipping songs. 

Ellipsize is already implemented, but it isn't strict enough to keep the buttons from jumping around.

Launchpad Details: [#LP1578440](https://bugs.launchpad.net/bugs/1578440) Zisu Andrei - 2016-05-05 00:01:21 +0000

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/43953529-buttons-jump-around-when-skipping-songs?utm_campaign=plugin&utm_content=tracker%2F59755748&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F59755748&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by nathanial**
Added a FixedWidthLabel that has a constant size, to avoid the buttons jumping around when songs of different title length get played. fixes #12

### CODE REVIEW

**Comment by nathanial** [comment]
@danrabbit Should I comment on the launchpad issue as well or are you guys moving to github issues exclusively?

**Comment by danrabbit** [comment]
@nathanial we're moving to GitHub exclusively :)

File: src/Widgets/FixedWidthLabel.vala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+public class FixedWidthLabel : Gtk.Label {
+    private int fixed_width;
+
+    public FixedWidthLabel(int fixed_width) {
+        this.fixed_width = fixed_width;
+        // Fix this when on gtk 3.16
+        this.set("xalign", 0);
```
Comment (donadigo) [inline src/Widgets/FixedWidthLabel.vala]: This should just be `xalign = 0`. elementary OS is currently on Gtk+3.18

**Comment by matzipan** [comment]
I'm closing this PR since it's been inactive and the solution is not what is desired.

**Pull request closed without merging**